### PR TITLE
add opticalglass and rayoptics python packages to conda-forge

### PR DESCRIPTION
**Pull request by mjhoptics**
Both of the submitted packages use pyqt5. It appears that conda-forge has an entry of pyqt that is actually pyqt5. Could you confirm or correct my understanding on this?
First time submitter. Constructive criticism is appreciated.
mjh

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/rayoptics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - qdarkstyle >=2.8
+    - qtconsole >=4.5.5
+    - requests >=2.23.0
+    - scipy >=1.3.1
+    - traitlets >=4.3.2
+    - transforms3d >=0.3.1
+
+test:
+  imports:
+    - rayoptics
```
Comment (chrisburr) [inline recipes/rayoptics/meta.yaml:44]: Is there a reason for not including pip check here?
```suggestion
    - rayoptics
  commands:
    - pip check
  requires:
    - pip
```

Comment (mjhoptics) [reply recipes/rayoptics/meta.yaml:44]: I think by removing pip check, I removed the symptom of a problem, but not the problem. With pip check, it flags a dependency of qdarkstyle, helpdev, as missing. This package is on pipy and hosted in GitLab but not on conda-forge or in anaconda.
I presume the correct solution would be for me to set up helpdev as a conda-forge submission. If there's another approach, please let me know.
Thank you for your help.

Comment (mjhoptics) [reply recipes/rayoptics/meta.yaml:45]: OK, I'm stuck now. I tried including helpdev and there's a message about conda-verify failing that I think prevents the build from completing. I tried reproducing on my local machine and conda-verify seems happy.
I admit I'm a little mystified that qdarkstyle is available on conda-forge but appears to depend on helpdev which isn't on conda-forge.

Comment (mjhoptics) [reply recipes/rayoptics/meta.yaml:45]: I've examined the use of helpdev by the qdarkstyle package. It is only used if you run the qdarkstyle package as a script to document the state of your environment, e.g. for a bug report. It is not used if you are using qdarkstyle programmically with pyqt, as is done with rayoptics (and probably all other packages depending on qdarkstyle).
I am going to remove the pip check test and helpdev from this PR.

Comment (chrisburr) [reply recipes/rayoptics/meta.yaml:45]: Indeed, this is a bug in the `qdarkstyle` recipe. I've made a PR to fix it and the "pip check" test can be re-added after https://github.com/conda-forge/qdarkstyle-feedstock/pull/14 is merged.

That said, there is no reason for it to hold up this PR so LGTM 👍

**Pull request merged**